### PR TITLE
K Closest Points to Origin

### PR DESCRIPTION
**Pull request by Chaitra-Bhat383**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ The solutions are located under `/LeetcodeProblems`. Each problem has a test fil
 | [Award Budget Cuts](/LeetcodeProblems/Algorithms/Award_Budget_Cuts.js)                                          |        | |
 | [Happy Number](https://leetcode.com/problems/happy-number/)                                                     | Easy   | https://leetcode.com/problems/happy-number/ |
 | [Shuffle String](https://leetcode.com/problems/shuffle-string/)                                                 | Easy   | https://leetcode.com/problems/shuffle-string/ |
+| [K Closest Points to Origin](/LeetcodeProblems/Algorithms/K_Closest_Points_to_Origin.js/)                       | Medium | https://leetcode.com/problems/k-closest-points-to-origin/
```
Comment (ignacio-chiazzo) [inline README.md:95]: nit: This list is sorted by hard/medium/easy, could you add it to the Medium section?

**Review by ignacio-chiazzo — COMMENTED** [review]
The problem/solution looks good to me. Could you fix the linter issue by running `npx eslint LeetcodeProblems LeetcodeProblemsTests --fix`? thanks!

File: README.md (hunk 1 above)
Comment (Chaitra-Bhat383) [reply README.md:95]: Yes yes I can do that!

**Comment by ignacio-chiazzo** [comment]
I just enabled the linter to run on this PR. You can also check lint errors locally by running `npx eslint LeetcodeProblems LeetcodeProblemsTests`

**Comment by Chaitra-Bhat383** [comment]
I resolved all the linter errors, please have a check!

File: README.md [hunk 2]
```diff
@@ -1,134 +1 @@
...
-I'd be pleased to accept contributions. I'd be happy to discuss problems and solutions over a Pull Request or an Issue.
-
-Each problem should have:
-
-1) A description of the problem at the top of the file.
-2) A test file with some test cases. The test file must export a `test()` function which should run all the tests of the file.
-3) An entry of the problem in the list of solutions within the README file.
-
-PR Example: https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/pull/39
+ [Maximal Square ](/LeetcodeProblems/Maximal_Square.js)                                               | Medium | https://leetcode.com/problems/maximal-square/
```
Comment (ignacio-chiazzo) [inline README.md:1]: Could you update the readme? It deleted all the previous staff.

**Review by ignacio-chiazzo — COMMENTED** [review]
I see some linting errors in `LeetcodeProblems/Algorithms/K_Closest_Points_to_Origin.js` file
Screen Shot 2022-10-23 at 10 45 32 AM

File: README.md (hunk 2 above)
Comment (Chaitra-Bhat383) [reply README.md]: Yes did it now!

**Review by ignacio-chiazzo — COMMENTED** [review]
oh I see what's going on. Could you rebase? A bad commit has been reverted and it caused to have two readmes. https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/pull/78

**Comment by Chaitra-Bhat383** [comment]
I hope its right now!

**Review by ignacio-chiazzo — COMMENTED** [review]
It looks good now, Could you rebase with the latest master so that I can hit rebase and merge and add you as a contributor?

**Comment by Chaitra-Bhat383** [comment]
Could you merge it now?

**Comment by ignacio-chiazzo** [comment]
Could you rebase with master? An easy way to do it would be to squash all your 10 commits into 1 and then rebase with the latest master. I can do it for you but Github won't mark you as contributor if I'm not mistaken.

Screen Shot 2022-10-23 at 3 43 35 PM

**Pull request closed without merging**